website/src/Menu.js

```diff
@@ -1,52 +1,131 @@
-function Links() {
   return {
     view: vnode => {
       return [
-        m(Link, {href:'/', id: 'home-link', ...vnode.attrs}, 'home'),
-        m(Link, {href:'/projects', id: 'projects-link', ...vnode.attrs}, 'projects'),
-        m(Link, {href: '/renders', id: 'renders-link', ...vnode.attrs}, 'renders'),
-        m(Link, {
-          href: User.username ? '/logout' : '/login',
-          id: 'login-link',
-          ...vnode.attrs
-        }, User.username ? 'logout' : 'login'),
+        m(
+          'header',
+          m('.navbutton', m('i.material-icons', { onclick: toggle }, 'menu')),
+          m('.navContainer', { style: { width: open ? '250px' : '0px' } }, [
+            m(navLinks, { onclick: toggle }),
+          ]),
+          //TODO: change logo to new image in the future when location of images is known
+          m(
+            'div.logo',
+            m(
+              Link,
+              { href: '/', id: 'home-link', ...vnode.attrs },
+              m('img.logoImg', { src: LOGO })
+            )
+          ),
+          m('.menuLinks', m(Links)),
+          m(
+            Link,
+            {
+              class: 'loginLink',
+              href: User.username ? '/logout' : '/login',
+              id: 'login-link',
+              ...vnode.attrs,
+            },
+            User.username ? 'Logout' : 'Login'
+          )
+        ),
       ]
-    }
+    },
   }
 }
 
+//an individual link
+export const Link = () => ({
+  view: vnode => {
+    return m(
+      m.route.Link,
+      {
+        class: window.location.pathname.startsWith(vnode.attrs.href)
+          ? 'active'
+          : '',
+        ...vnode.attrs,
+      },
+      vnode.children
+    )
+  },
+})
+
+//the links in the menu bar
+export const Links = () => ({
+  view: vnode => [
+    m(
+      Link,
+      {
+        class: 'menuLink',
+        href: '/projects',
+        id: 'projects-link',
+        ...vnode.attrs,
+      },
+      'PROJECTS'
+    ),
+    m(
+      Link,
+      {
+        class: 'menuLink',
+        href: '/renders',
+        id: 'renders-link',
+        ...vnode.attrs,
+      },
+      'RENDERS'
+    ),
+  ],
+})
 
-export { Menu, Links, Link }
+//the links in the nav bar, separated to have own style
+export const navLinks = () => ({
+  view: vnode => [
+    m(
+      Link,
+      { class: 'navLink', href: '/', id: 'home-link', ...vnode.attrs },
+      'HOME'
+    ),
+    m(
+      Link,
+      {
+        class: 'navLink',
+        href: '/projects',
+        id: 'projects-link',
+        ...vnode.attrs,
+      },
+      'PROJECTS'
+    ),
+    m(
+      Link,
+      {
+        class: 'navLink',
+        href: '/renders',
+        id: 'renders-link',
+        ...vnode.attrs,
+      },
+      'RENDERS'
+    ),
+    m(
+      Link,
+      {
+        class: 'navLink',
+        href: User.username ? '/logout' : '/login',
+        id: 'login-link',
+        ...vnode.attrs,
+      },
+      User.username ? 'LOGOUT' : 'LOGIN'
+    ),
+  ],
+})
```

Comment:
For this PR, I'm fine with having a separate `navLinks` component, but in a future refactor, we should remove the duplication from the `Links` component. 

If the only difference is the `navLinks` have the `.navLink` class, then it should be a simple inheritance solution. But that all can wait until a future PR.

Reply:
awesome! The later Home redesign also removes the menu and links from the Home.js. But good to update it here with the menu changes. Thank you sir